lib/standalone-tests/terminus.nofailuresduringshutdown.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+const http = require('http')
+const server = http.createServer((req, res) => res.end('hello'))
+
+const { createTerminus } = require('../../')
+
+createTerminus(server, {
+  healthChecks: {
+    '/health': () => Promise.resolve()
+  },
+  sendFailuresDuringShutdown: false,
+  onSendFailureDuringShutdown: async () => {
+    console.log('onSendFailureDuringShutdown')
```

Comment:
did you mean to assert this string in the test cases?

Reply:
No, I'm printing it to make sure it's _not_ logged - since we're not sending failure responses during shutdown, this method should never be called.

In `terminus.spec.js` on line 355, we're asserting that stdout is an empty string.

Let me know if you want this changed to something else!

Reply:
my bad, thank you